### PR TITLE
feat: include post type and source private fields in tinybird export

### PR DESCRIPTION
**Pull request by vpol**
Signed-off-by: Viktor Poluksht <viktor@daily.dev>

### CODE REVIEW

File: src/cron/exportToTinybird.ts [hunk 1]
```diff
@@ -22,16 +22,17 @@ const cron: Cron = {
...
-              ("banned" or "deleted") ::int AS "banned"
+              "authorId"                    AS "author_id",
+              "createdAt"                   AS "created_at",
+              "metadataChangedAt"           AS "metadata_changed_at",
+              "creatorTwitter"              AS "creator_twitter",
+              "sourceId"                    AS "source_id",
+              "tagsStr"                     AS "tags_str",
+              ("banned" or "deleted") ::int AS "banned",
+              "type"                        AS "post_type",
+              "private"                     AS "source_private"
```
Comment (idoshamun) [inline src/cron/exportToTinybird.ts:33]:
```suggestion
              "private"                     AS "post_private"
```

File: src/cron/exportToTinybird.ts [hunk 2]
```diff
@@ -22,16 +22,17 @@ const cron: Cron = {
...
-              ("banned" or "deleted") ::int AS "banned"
+              "authorId"                    AS "author_id",
+              "createdAt"                   AS "created_at",
+              "metadataChangedAt"           AS "metadata_changed_at",
+              "creatorTwitter"              AS "creator_twitter",
+              "sourceId"                    AS "source_id",
+              "tagsStr"                     AS "tags_str",
+              ("banned" or "deleted") ::int AS "banned",
+              "type"                        AS "post_type",
+              "private"::int                AS "source_private"
```
Comment (idoshamun) [inline src/cron/exportToTinybird.ts:33]: @vpol I think we changed the name of this column right?

Comment (vpol) [reply src/cron/exportToTinybird.ts]: yes, fixed. forgot about it.

**Review by idoshamun — APPROVED** [review]
Please verify that it works after the deployment

**Pull request merged**